#!/bin/sh /usr/share/dpatch/dpatch-run
## 202-magic-update-blender.dpatch by Josh Babcock <jbabcock@atlantech.net>
##
## DP: Adds magic for Blender python scripts (Closes: #323291).

@DPATCH@

diff -Naur file-4.20.orig/magic/Magdir/blender file-4.20/magic/Magdir/blender
--- file-4.20.orig/magic/Magdir/blender	2000-08-05 17:12:09.000000000 +0000
+++ file-4.20/magic/Magdir/blender	2007-03-27 16:28:58.000000000 +0000
@@ -15,3 +15,5 @@
 >9	byte	x		with version %c.
 >10	byte	x		\b%c
 >11	byte	x		\b%c
+
+0	string	#!BPY		Blender3D Python file
